src/status_im/common/biometric/events.cljs

```diff
@@ -28,6 +29,12 @@
     :FaceID      (i18n/label :t/biometric-faceid)
     (i18n/label :t/biometric-touchid)))
 
+(defn get-icon-by-type
+  [biometric-type]
+  (case biometric-type
```

Comment:
we generally prefer `condp =` as `case` does not pick up compile time errors, in this case you can probably just use an `(if`, though I take the idea is that we will have more eventually

Reply:
yes, in case we would like to differentiate touch id with fingerprint or the face biometrics on android in the future